Tensile/TensileCreateLibrary.py

```diff
@@ -164,6 +166,10 @@ def getAssemblyCodeObjectFiles(kernels, kernelWriterAssembly, outputPath):
                     print(err.output)
                     raise
 
+                #if removeTemporaries:
```

Comment:
This doesn't have a significant impact on the high-water mark and is breaking when targeting gfx90a.